woocommerce/woocommerce-android
# [Mobile Payments] Handle stripe requirements overdue with chrome

Closes: #9686


### Description

The PR adds handling of the Stripe Account Requirements Overdue error. We open the same payments setup screen on the Wp-admin in this case

### Testing instructions

To test this:
* Create a new wpcom site
* Start your WC store (it will automatically install WCPAY)
* On the page `/wp-admin/admin.php?page=wc-admin&path=%2Fpayments%2Fconnect` click "finish setup"
* Enter any email -> This way the Stripe account will be created without data for KYC, which means it's in "restricted" status
* Now go to the app -> more -> Payments -> click on the Error
* Notice 2 CTAs. Try both
* The primary will lead you to the page where you can finish onboarding and provide all the info needed for Stripe

Also validate the tracking:


```
Tracked: card_present_onboarding_cta_tapped, Properties: {"reason":"stripe_account_setup_tapped","plugin_slug":"woocommerce-payments","country":"unknown","blog_id":223230062,"is_wpcom_store":true,"was_ecommerce_trial":false,"plan_product_slug":"business-bundle","is_debug":true,"site_url":"https:\/\/withoutwcpaysetup2.blog"}
```

### Images/gif

https://github.com/woocommerce/woocommerce-android/assets/4923871/b5c4382f-b76a-44b7-b780-1ce655dbde8c
